tests/test_smc_reader.py

```diff
@@ -108,46 +91,82 @@ def test_get_iphone_intrinsics():
 
 
-def test_get_iphone_data():
+def test_get_iphone_color():
     smc = SMCReader(TEST_SMC_PATH)
 
-    iphone_color = smc.get_iphone_color(iphone_id=0, frame_id=0)
-    assert iphone_color.shape == (
-        1, 1440, 1920, 3), 'iPhone color image should have shape 1x1440x1920x3'
+    with pytest.raises(AssertionError):
+        _ = smc.get_color(device='iphone', device_id=0, frame_id=0)
+
+    with pytest.raises(KeyError):
+        _ = smc.get_color(device='iPhone', device_id=1, frame_id=0)
+
+    iphone_color = smc.get_iphone_color(iphone_id=0, frame_id=0, vertical=True)
+    assert iphone_color.shape == (1, 1920, 1440, 3), \
+        'iPhone color image in vertical mode ' \
+        'should have shape 1x1920x1440x3.'
 
-    iphone_depth = smc.get_iphone_depth(iphone_id=0, frame_id=0)
-    assert iphone_depth.shape == (
-        1, 192, 256), 'iPhone depth image should have shape 1x192x256'
+    iphone_color = smc.get_iphone_color(
+        iphone_id=0, frame_id=0, vertical=False)
+    assert iphone_color.shape == (1, 1440, 1920, 3), \
+        'iPhone color image in horizontal mode ' \
+        'should have shape 1x1440x1920x3.'
 
 
-def test_get_color():
+def test_get_iphone_depth():
     smc = SMCReader(TEST_SMC_PATH)
 
-    kinect_color = smc.get_color(device='Kinect', device_id=0, frame_id=0)
-    assert kinect_color.shape == (
-        1, 1080, 1920,
-        3), 'Kinect Color should have resolution of 1x1080x1920x3'
+    with pytest.raises(KeyError):
+        _ = smc.get_iphone_depth(iphone_id=1, frame_id=0)
 
-    iphone_color = smc.get_color(device='iPhone', device_id=0, frame_id=0)
-    assert iphone_color.shape == (
-        1, 1440, 1920, 3), 'iPhone color image should have shape 1x1440x1920x3'
+    iphone_depth = smc.get_iphone_depth(iphone_id=0, frame_id=0, vertical=True)
+    assert iphone_depth.shape == (1, 256, 192), \
+        'iPhone depth image in vertical mode should have shape 1x256x192.'
+
+    iphone_depth = smc.get_iphone_depth(
+        iphone_id=0, frame_id=0, vertical=False)
+    assert iphone_depth.shape == (1, 192, 256), \
+        'iPhone depth image in horizontal mode should have shape 1x192x256.'
 
-    with pytest.raises(AssertionError):
-        _ = smc.get_color(device='kinect', device_id=0, frame_id=0)
+
+def test_get_kinect_color():
+    smc = SMCReader(TEST_SMC_PATH)
 
     with pytest.raises(AssertionError):
-        _ = smc.get_color(device='iphone', device_id=0, frame_id=0)
+        _ = smc.get_color(device='kinect', device_id=0, frame_id=0)
 
     with pytest.raises(KeyError):
         _ = smc.get_color(device='Kinect', device_id=10, frame_id=0)
 
-    with pytest.raises(KeyError):
-        _ = smc.get_color(device='iPhone', device_id=1, frame_id=0)
+    kinect_color = smc.get_color(device='Kinect', device_id=0, frame_id=0)
+    assert kinect_color.shape == (1, 1080, 1920, 3), \
+        'Kinect Color should have resolution of 1x1080x1920x3'
+
+
+def test_get_kinect_depth():
```

Comment:
Maybe rename this test?

Reply:
Rearranged in https://github.com/open-mmlab/mmhuman3d/pull/76/commits/93bf306e0e9ea77c01a8d7d349f79f1e67dd53ae.